main.js

```diff
@@ -0,0 +1,51 @@
+console.log('Let it snow, let it snow, let it snow!');
+
+// create querySelectors
+var button = document.querySelector('#add-expense');
+var expenseTable = document.querySelector('tbody');
+var inputExpenseName = document.querySelector('#expense-name');
+var inputExpenseAmount = document.querySelector('#expense-amount');
+var expenseTotal = document.querySelector('.expense-total');
+
+var handleInput = function(event) {
+  if (event.keyCode === 13);
```

Comment:
You're not branching off of this `if` statement. Without curly braces to introduce a block nothing happens.